noisy_simulator/src/instrument.rs

```diff
@@ -102,7 +108,9 @@ fn summed_kraus_operators(operations: &[Operation]) -> Result<Vec<SquareMatrix>,
                 .iter()
                 .map(|k| {
                     // This code is doing the equivalent to:
-                    // choi_matrix += vec(K) * vec(K)^dagger
+                    // choi_matrix += vectorized(K) * vectorized(K).adjoint()
+                    // Note that if you multiply a row vector times a column
```

Comment:
Is this back-to-front? I thought if you multiple a column vector by a row vector you get a matrix (i.e, a density matrix is `|ψ><ψ|`), but a row vector times a column vector is like an inner product and gives you a scalar.

Reply:
Ah, yes, you are right! Nice catch.